srcpkgs/qalculate-gtk/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'qalculate-gtk'
 pkgname=qalculate-gtk
-version=3.5.0
+version=3.6.0
 revision=1
 build_style=gnu-configure
 hostmakedepends="intltool pkg-config"
-makedepends="libqalculate-devel gtk+3-devel"
+makedepends="libqalculate-devel>=3.5.0 gtk+3-devel>=3.10"
```

Comment:
why these restrictions?

Reply:
Because of this: [link](https://qalculate.github.io/downloads.html)(#Graphical user interface requirements)

Reply:
alright then. It's just pretty unusual to restrict makedepends like this.